### DISCUSSION TITLE
如何更改Menu收起状态下，弹出的子菜单样式

### DISCUSSION THREAD

**Discussion by ghost** [ant-design/ant-design#37647] [Q&A]
### What problem does this feature solve?

可以个性化定制弹出子菜单样式

### What does the proposed API look like?

目前弹出的菜单是直接挂载在body上，无法对其添加样式进行修改

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/70744950/191159556-f99f23d0-4134-4203-bcf0-5b5c69b7cf27.png">

**Comment by MadCcc** [comment]
如果是想要子菜单亮色模式，SubMenu 也提供了 `theme` 属性。
想要魔改的话在 body 上也是可以的。

**Reply by ghost** [reply]
目前需求是想要在body上进行魔改，但是找不到可以添加弹出菜单className的入口，在项目代码中，对Menu添加一个className，但是因为弹出的菜单是采用的定位，无法使用添加的className找到弹出菜单的className。

**Reply by SignDawn** [reply]
popupClassName

**Reply by ghost** [reply]
popupClassName是在mode不等于inline时，可以失效，目前需求是mode = 'inline'时，在菜单收起时，更改弹出的样式

**Comment by bleackking** [comment]
我在自定义收缩菜单的时候，点击收缩菜单后，那个弹出的子菜单不会消失，这是什么问题
